CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This release does not support any new features added in API versions 800,1000 an
 - Golden Image
 - OS Volume
 - Plan Script
+- Hypervisor Manager
```

Comment:
please add it in a new unreleased version as this has been already released

Reply:
@soodpr Changes Updated

Reply:
its still present

Reply:
Its removed now, please check.